src/AccessibilityInsights.SharedUx/Controls/DisplayCountControl.xaml

```diff
@@ -10,6 +10,6 @@
              d:DesignHeight="300" d:DesignWidth="300"
              AutomationProperties.Name="{x:Static Properties:Resources.DisplayCountControlAutomationName}">
     <Grid>
-        <TextBlock x:Name="lbCount" FontSize="100" HorizontalAlignment="Center" VerticalAlignment="Center" Foreground="Blue" Text="5" FontWeight="Bold" Background="Transparent" AutomationProperties.LiveSetting="Polite"/>
+        <TextBlock x:Name="lbCount" FontSize="100" HorizontalAlignment="Center" VerticalAlignment="Center" Foreground="Blue" Text="{x:Static Properties:Resources.DisplayCount}" FontWeight="Bold" Background="Transparent" AutomationProperties.LiveSetting="Polite"/>
```

Comment:
I wonder if we can just remove this Text completely? The value gets dynamically set at https://github.com/microsoft/accessibility-insights-windows/blob/main/src/AccessibilityInsights.SharedUx/Controls/DisplayCountControl.xaml.cs#L26